### PR TITLE
Ft/update get tests

### PR DESCRIPTION
**Pull request by electrachong**
Depends on https://github.com/scality/Arsenal/pull/303

We weren't testing the response content headers for objectGet, so we did not catch when it was not working with v4 auth.

### CODE REVIEW

**Comment by electrachong** [comment]
@ironman-machine 
SCALITY_ARSENAL_BRANCH=fix/response-content-headers

File: tests/functional/aws-node-sdk/test/object/get.js [hunk 1]
```diff
@@ -57,36 +59,79 @@ describe('GET object', () => {
...
-                      return done();
-                  });
+
+            describe('if response content headers are set in query', () => {
+                before(done => {
+                    s3.putObject({ Bucket: bucketName, Key: objectName },
+                        err => done(err));
+                });
+
+                it('should return additional headers even if not set in ' +
```
Comment (LaurenSpiegel) [inline tests/functional/aws-node-sdk/test/object/get.js]: aren't you setting them through the params?

Comment (electrachong) [inline tests/functional/aws-node-sdk/test/object/get.js]: yep -- but they are set in the objectGet request, not the objectPut request (see next line), in contrast to the previous test

**Comment by electrachong** [comment]
after fixes
@ironman-machine
SCALITY_ARSENAL_BRANCH=fix/response-content-headers

**Pull request merged**